cassandra-mesos-executor/src/main/java/io/mesosphere/mesos/frameworks/cassandra/executor/CassandraExecutor.java

```diff
@@ -160,12 +160,14 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo task) {
                         serverTask = task;
                         jmxConnect = objectFactory.newJmxConnect(cassandraServerRunTask.getJmx());
                         startCheckingHealth(driver, jmxConnect, cassandraServerRunTask.getHealthCheckIntervalSeconds());
-                        driver.sendStatusUpdate(taskStatus(serverTask, TaskState.TASK_RUNNING,
-                            SlaveStatusDetails.newBuilder()
-                                .setStatusDetailsType(SlaveStatusDetails.StatusDetailsType.CASSANDRA_SERVER_RUN)
-                                .setCassandraServerRunMetadata(CassandraServerRunMetadata.newBuilder()
-                                    .setPid(process.getPid()))
-                                .build()));
```

Comment:
Makes sense. I had it just call serverProcessNotRunning() without the return call, so that a log call at the end of the function is still reached.